libs/server/user/api-rest/src/lib/controllers/user.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, UseGuards } from '@nestjs/common';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UserFacade } from '@pimp-my-pr/server/user/core/application-services';
+import { GetUserInfoQuery } from '@pimp-my-pr/server/user/core/application-services';
+import { UserReadModel } from '@pimp-my-pr/shared/domain';
+import { AuthGuard, CurrentUserId } from '@pimp-my-pr/server/auth/public';
+
+@ApiTags('user')
+@ApiBearerAuth()
+@UseGuards(AuthGuard)
+@Controller('user-info')
+export class UserController {
+  constructor(private userFacade: UserFacade) {}
+
+  @Get()
+  getUserInfo(@CurrentUserId() currentUserId: string): Promise<UserReadModel> {
```

Comment:
we need a dedicated model class that will be returned from the controller to decorate its props for a swagger. Check out RepositoriesStatisticsItemReadModel.